apps/react-native/babel.config.js

```diff
@@ -0,0 +1,33 @@
+module.exports = function(api) {
+  api.cache(true);
+
+  const presets = [
+    "module:metro-react-native-babel-preset",
+    [
+      "@babel/preset-env", {
```

Comment:
what was your decision for using `@babel/preset-env`?

Reply:
I got an error during development that looked like it was caused by this being missing.  This preset was already present in package.json in the example app.

Reply:
removed @babel/preset-env plugin